tinker_cookbook/rl/train.py

```diff
@@ -229,6 +229,7 @@ class Config:
     dataset_builder: RLDatasetBuilder  # also determines batch size
     model_name: str
     max_tokens: int
+    temperature: float = 1.0
```

Comment:
Hi @Xiuyu-Li, before I hit merge, could you please add a comment here noting that changing it is not recommended, because the training algorithm might not behave correctly under all settings (e.g., compute_logprobs doesn't currently accept a temperature parameter, so if you're using KL penalty, it won't be computed correctly).

Reply:
I added this myself